### ISSUE TITLE
Is github entreprise supported ?

### ISSUE THREAD

**Issue by tristanbes** [wip/app#92]
Hello,

We're soon migrating from Github SAAS to Github Entreprise. 
Is WIP app supported on Gtihub Entreprise ?

Thank you :-)

**Comment by gr2m** [comment]
I think so, as long as you use the latest version of WIP. I also created a docker image but that part is new to me so might be a little outdated. Please let me know if there is anything I can do to make it easier for you.

I want to look into using replicated.com for the enterprise version. It will be either free or I’ll donate all revenue for it to https://railsgirlssummerofcode.org

**Comment by tristanbes** [comment]
> I think so, as long as you use the latest version of WIP
You mean a local installation of WIP should do it right ?

Thanks for the answer

**Comment by bbatha** [comment]
It works on our 2.13 instance.

**Comment by gr2m** [comment]
@bbatha how do you set it up locally? I wonder if there is something I can do to make that process easier? I was looking into creating a docker image and maybe setup https://www.replicated.com/, but I don’t have experience with any of it. Do you have any recommendations?

**Comment by bbatha** [comment]
We have an internal heroku like service that can use heroku buildpacks. We're just using the standard node buildpack through the auto-detection features to deploy it. We didn't need to do anything special.

I've installed a few tools using replicated, its a fairly easy experience as a consumer. I've never bundled anything with it. But it might be overkill for an opensource project that's just a single small service. A simple docker container is probably sufficient for this.

**Comment by tristanbes** [comment]
So, would it be possible to have installation steps to guide users/adminsys please ? I'm confused here.

**Comment by gr2m** [comment]
I’d love help to create a docker image for the app, I’ve no idea about containers :D

**Comment by gr2m** [comment]
I think it would make a great blog post: "How to package your probot app for GitHub Enterprise". 

It sounds like you do it for your own GHE instance. For WIP, I’d be more interested in providing the most simple way possible to run WIP with their GHE instance. I don’t know what the best practise is here, I assume it’s a Docker image, but I don’t know. And if it is a Docker image, what would it look like to help with the setup, etc

**Comment by HaydenMeloche** [comment]
Yeah, I and a few others built our own little probot app and that's the process I went through to get it working. 

Since the automated way isn't supported a Docker image wouldn't make the setup any easier I feel. Someone would still have to manually setup the app on GHE and fill out the env file. But it would make deployment easier & you could pass the above info into docker.
https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file

**Comment by gr2m** [comment]
If anyone is interested, I'd love a Dockerfile and documentation on how to deploy the WIP for GitHub Enterprise Server. I don't have a GHES instance available myself for testing
